model/common_structures.smithy

```diff
@@ -59,3 +59,12 @@ structure Hits{
     _source: Document,
     fields: Document
 }
+
+structure PitsDetailsDeleteAll{
+    successful: Boolean,
+    pit_id: String
+}
+
+list PitsDeleteAll{
```

Comment:
Both of these can be added to the structures.smithy file of the API itself since they are not being used across other API models.